octree_web_viewer/client/octree_viewer.ts

```diff
@@ -79,7 +81,30 @@ class NodeRenderData {
 }
 
 class NodeLoader {
+
+  private batchSize = 50;
+  private maxConcurrentRequests = 2;
+  private queue = new AsyncTasksQueue<void>(this.maxConcurrentRequests);
+
   public load(
+    scene: THREE.Scene,
+    material: THREE.ShaderMaterial,
+    nodes: NodeData[],
+    onNewNodeData: () => void
+  ): Promise<void[]> {
```

Comment:
Is there a reason to return `void[]`? Assuming I understand this correctly, something like the following might be cleaner:

```
public async load(/* ... */) {
  await Promise.all(/* ... */);
}
```